examples_tf2_py/examples_tf2_py/static_broadcaster.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import math
+
+from geometry_msgs.msg import TransformStamped
+import rclpy
+from rclpy.node import Node
+from tf2_ros.static_transform_broadcaster import StaticTransformBroadcaster
+
+
+class StaticFramePublisher(Node):
+    """
+    Broadcast transforms that never change.
+
+    This example publishes transforms from `base_link` to a platform and sensor
+    frames.
+    The transforms are only published once at startup, and are constant for all
+    time.
+    """
+
+    def __init__(self):
+        super().__init__('example_static_frame_publisher')
+
+        self._tf_publisher = StaticTransformBroadcaster(self)
+
+        # Publish static transforms once at startup
+        self._tf_publisher.sendTransform(self.make_transforms())
+
+    def make_transforms(self):
+        base_link_to_platform = TransformStamped()
+        base_link_to_platform.header.frame_id = "base_link"
```

Comment:
This is a question: should we initialize `header.stamp` in the message?

Reply:
I guess this should. The `static_transform_broadcaster` CLI tool sets `header.stamp`: https://github.com/ros2/geometry2/blob/71f823e90b699c9535ce682db777f7bc88cdfbcd/tf2_ros/src/static_transform_broadcaster_program.cpp#L88

Reply:
Set stamp on static transforms in 8de82bd